src/audio/module_adapter/module/waves.c

```diff
@@ -693,7 +684,7 @@ static int waves_codec_prepare(struct processing_module *mod)
 	struct comp_dev *dev = mod->dev;
 	int ret;
 
-	comp_dbg(dev, "waves_codec_prepare() start");
+	comp_info(dev, "waves_codec_prepare() start");
```

Comment:
why do you need all these debug prints? I'd really avoid adding too many of them